pkg/types/types.go

```diff
@@ -54,6 +54,9 @@ var ReadyLogMessageByRole = map[Role]string{
 	LoadBalancerRole: "start worker processes",
 }
 
+// ContainerRestartingStatus defines the status string that signals the node container is restarting
+var ContainerRestartingStatus = "restarting"
```

Comment:
```suggestion
// NodeStatusRestarting defines the status string that signals the node container is restarting
const NodeStatusRestarting = "restarting"
```
Outside of the runtime level, we're referring to them as nodes.. not critical though :man_shrugging: